sources/models/src/vmware-dev/override-defaults.toml

```diff
@@ -0,0 +1,12 @@
+[configuration-files.containerd-config-toml]
+# No override to path
+template-path = "/usr/share/templates/containerd-config-toml_aws-dev"
```

Comment:
Can we make a vmware-dev file instead?  If it's the same, it could be a symlink in the packages directory, but having the wrong variant name on the system at runtime feels wrong...

Reply:
I agree.  A symlink seems fine.

Reply:
I couldn't use a symlink for this file and still reference it in the `containerd.spec` so I opted to just create an actual file.

Reply:
We should be able to use a symlink; what happened when you tried?  Can we dig in a bit?  If you post or share the changes you tried, I bet we can figure it out.

Reply:
Either way we should update the path here to point to `containerd-config-toml_vmware-dev`.